public/app/core/components/NavBar/Next/NavBarItem.tsx

```diff
@@ -81,7 +83,7 @@ const NavBarItem = ({
     );
   } else {
     return (
-      <li className={cx(styles.container, className)}>
+      <li className={cx(styles.container, section.id === menuIdOpen && styles.containerHover, className)}>
```

Comment:
idk if this makes a difference but i think this is the preferred syntax for conditional formatting in emotion:
```suggestion
      <li className={cx(styles.container, { [styles.containerHover]: section.id === menuIdOpen }, className)}>
```

the current way works because `cx` just ignores falsey values. i'm guessing there's a slight performance benefit to doing it properly but honestly idk 😂

Reply:
Thanks for raising! It did look a bit ugly, but wasn't sure how `cx` expected it normally. I'll commit your change